ppl/src/main/antlr/OpenSearchPPLLexer.g4

```diff
@@ -7,7 +7,7 @@
 lexer grammar OpenSearchPPLLexer;
 
 channels { WHITESPACE, ERRORCHANNEL }
-
+options { caseInsensitive = true; }
```

Comment:
Just wonder is this only for consistency with PPL Spark grammar? Because we already have `CaseInsensitiveCharStream` in this repo?

Reply:
Issue #3391 mentioned that user uses the g4 files in frontend (to generated js parser for example). So we should fix the case sensitive issue in g4 self-contained. @dai-chen